examples/visualization/styles/helpers/size-bins-style.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css"
+    />
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.1.0/dist.min.js"></script>
+
+    <!-- CARTO Deck.gl's Tile Layer -->
+    <script src="/packages/toolkit/dist/umd/index.min.js"></script>
+
+    <script>
+      async function initialize() {
+        carto.setDefaultCredentials({ username: 'public' });
+
+        const deckMap = new window.deck.DeckGL({
+          container: 'map',
+          mapStyle:
+            'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+          initialViewState: {
+            longitude: -3.700385991742541,
+            latitude: 40.41532659629741,
+            zoom: 14
+          },
+          controller: true
+        });
```

Comment:
Right now, with map helper, you can use something like:

```js
const deckMap = carto.viz.createMap({ view: { zoom: 4 } });
```
It will use 'voyager' basemap by default and container: map, but you can add / tweak any parameter